containerized_ml_models/api/model.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+import pickle as pkl
+import numpy as np
+
+
+class Model:
+
+    __create_key = object()
+    class_instance = None
+
+    def __init__(self, model_path: str, create_key: object):
+        if create_key is not self.__create_key:
+            raise ValueError(
+                "You shouldn't call __init__ method. "
+                "Call Model.get_or_create instead"
+            )
+        with open(model_path, "rb") as f:
+            self.model = pkl.load(f)
```

Comment:
Always use descriptive names. Just `file` would be good

Reply:
Fixed